erizo/src/erizo/rtp/SenderBandwidthEstimantionHandler.cpp

```diff
@@ -50,33 +56,47 @@ void SenderBandwidthEstimationHandler::notifyUpdate() {
   if (!connection_) {
     return;
   }
+  updateMaxListSizes();
   stats_ = pipeline->getService<Stats>();
   if (!stats_) {
     return;
   }
   initialized_ = true;
 }
 
+void SenderBandwidthEstimationHandler::updateMaxListSizes() {
+  size_t num_streams = 0;
+  connection_->forEachMediaStream([&num_streams] (std::shared_ptr<MediaStream> media_stream) {
```

Comment:
Since we are only taking into account video packets for `packets_sent`, shouldn't we check we're only using video ssrcs?
I'm not sure if we did that properly before though, we can discuss this.

Reply:
good catch! I think it makes sense to check video and audio for RTP and RR packets.